suite-common/connect-popup/src/connectPopupThunks.ts

```
@@ -0,0 +1,181 @@
+    CustomThunkAPI
+> => connectPopupCallThunkInner(params) as any;
+
+export const connectPopupDeeplinkThunk = createThunk<void, { url: string }>(
+    `${CONNECT_POPUP_MODULE}/deeplinkThunk`,
+    async ({ url }, { dispatch }) => {
+        try {
+            const parsedUrl = new URL(url);
+            const path = parsedUrl.pathname;
+            const queryParams = Object.fromEntries(parsedUrl.searchParams.entries());
+
+            const version = path && path.split('/').slice(-2, -1)[0];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)